Makefile

```diff
@@ -195,3 +195,18 @@ generate-sudo-list:
 
 test-sudo-list:
 	@$(PYTHON) scripts/generate_sudo_list.py compare "docs/sudo_cmd_list.txt" || exit 1
+
+# shamelessly copied/modified from https://github.com/RedHatInsights/insights-host-inventory/blob/64fa5698c88cfb92258c532cfbd26a657faa07f6/Makefile#L48-L57
+# hence this role follows GPLv2 license (I guess... HELP)
```

Comment:
😅

Reply:
help

Reply:
this is the closest I found for an answer, but it's for stack overflow/exchange/&friends https://meta.stackexchange.com/a/371460

Reply:
After clarifying this with @kahowell, I believe this is fine =D